app/src/main/java/org/oppia/app/drawer/NavigationDrawerFragmentPresenter.kt

```diff
@@ -97,16 +98,21 @@ class NavigationDrawerFragmentPresenter @Inject constructor(
     if (previousMenuItemId != menuItemId && menuItemId != 0) {
       when (NavigationDrawerItem.valueFromNavId(menuItemId)) {
         NavigationDrawerItem.HOME -> {
+          drawerLayout.closeDrawers()
```

Comment:
For now, please don't bring any change to other options in nav drawer, just focus on "OPTIONS" only, because they are not working correctly.